wedsite/settings.py

```diff
@@ -422,35 +435,58 @@
 		),
 	},
 	"gifts" : GIFT_OPTIONS,
-	"landing_image" : LANDING_IMAGE,
+	"landing_page" : LANDING_PAGE,
 	"rsvp" : {
 		"cutoff" : RSVP_CUTOFF_DATE,
 		"active" : RSVP_ACTIVE,
 		"meal_description" : MEAL_DESCRIPTION,
+                "comments_prompt": "Please use the section below to add a general comment to your response.",
+                # NB these fields must correspond to acutal columns. you can
+                # relabel/filter them, but don't add new ones without changing
+                # the RSVPPerson model as well
+                "fields": {
+                    "name": "Name",
+                    "is_attending_rehearsal": "Attend Rehearsal Dinner?",
+                    "is_attending_wedding": "Attend Wedding?",
+                    "is_child": "Child?",
+                    "dietary_vegetarian": "Vegetarian",
+                    "dietary_vegan": "Vegan",
+                    "dietary_kosher": "Kosher",
+                    "dietary_gluten_free": "Gluten Free",
+                    "dietary_other": "Other Dietary Restriction",
+                    "special_requests": "Special Request",
+                },
 	},
 	"story" : STORY_ITEMS,
-	"credits" : CREDITS_ITEMS,
+	"credits" : {
+            "text": CREDITS_TEXT,
+            "people": CREDITS_ITEMS,
+        },
 	"traditions" : TRADITIONS_ITEMS,
 	"travel" : {
 		"airport_info" : AIRPORT_INFO,
 		"hotels" : HOTEL_ITEMS,
-		"airbnb" : AIRBNB_LINK,
-	},
-	"favicon" : FAVICON
+		"airbnb_link" : AIRBNB_LINK,
+	},
+	"favicon" : FAVICON,
+        "admin_email": ADMIN_EMAIL,
+        # Which pages have restricted access
+        #   Valid values are 'all', 'login', and False for no access at all
+        #   False may be desired if you don't want to use a page at all for the duration
+        #   of your wedding, or if it simply doesn't exist yet (e.g. for save the dates)
+        "access": {
+            "index" : "all",
```

Comment:
could use an `Enum` for this instead of strings (forgive me... i mainly write C :p)